Docker/Invoke-Build.ps1

```diff
@@ -0,0 +1,199 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
This duplicates https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/blob/main/Docker/build-images.ps1

I think we need to refactor both scripts, but that can be done in another ticket

Reply:
removed Build images ps1 file and  now all places invoked by invoke build ps1 file